content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -0,0 +1,346 @@
+---
+reviewers:
+- hasheddan
+- pjbgf
+- saschagrunert
+title: Seccomp
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+Seccomp stands for secure computing mode and has been a feature of the Linux
+kernel since version 2.6.12.  It can be used to sandbox the privileges of a
+process, restricting the calls it is able to make from userspace into the
+kernel. Kubernetes lets you automatically apply seccomp profiles loaded onto a
+Node to your Pods and containers.
+
+Identifying the privileges required for your workloads can be difficult. In this
+tutorial, we will go through how to load seccomp profiles into a local
+Kubernetes cluster, how to apply them to a Pod, and how you can begin to craft
+profiles that give only the necessary privileges to your container processes.
+
+## {{% heading "objectives" %}}
+
+* Learn how to load seccomp profiles on a node
+* Learn how to apply a seccomp profile to a container
+* Observe auditing of syscalls made by a container process
+* Observe behavior when a missing profile is specified
+* Observe a violation of a seccomp profile
+* Learn how to create fine-grained seccomp profiles
+* Learn how to apply a container runtime default seccomp profile
+
+## {{% heading "prerequisites" %}}
+
+In order to complete all steps in this tutorial, you must install
+[kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and
+[kubectl](../../tasks/tools/install-kubectl). We will show examples with both
```

Comment:
```suggestion
[kubectl](/doc/tasks/tools/install-kubectl/). This tutorial both
```
([avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we))